java/pom.xml

```diff
@@ -73,6 +68,16 @@
 
     </dependencies>
 
+    <repositories>
+        <repository>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <id>dremio-public</id>
+            <url>https://repo.drem.io/content/repositories/internal-releases/</url>
```

Comment:
Are you sure that we can provide links to dremio internal repositories in the public repo?

Reply:
Not needed as I now use public repo available version.